x-pack/legacy/plugins/fleet/common/constants/index_names.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 export const INDEX_NAMES = {
-  FLEET: '.fleet',
+  FLEET: '.kibana-fleet',
```

Comment:
Why the change here?

Reply:
It's not possible to launch kibana here, the kibana user has only access to `.kibana-*` indexes